tensorboard/webapp/metrics/views/metrics_container_test.ts

```diff
@@ -15,22 +15,53 @@ limitations under the License.
 import {NO_ERRORS_SCHEMA} from '@angular/core';
 import {TestBed} from '@angular/core/testing';
 import {By} from '@angular/platform-browser';
+import {Store} from '@ngrx/store';
+import {MockStore} from '@ngrx/store/testing';
+import {State} from '../../app_state';
+import {getEnableHparamsInTimeSeries} from '../../feature_flag/store/feature_flag_selectors';
+import {RunsSelectorContainer} from '../../runs/views/runs_selector/runs_selector_container';
+import {provideMockTbStore} from '../../testing/utils';
 import {MetricsDashboardContainer} from './metrics_container';
 
 describe('metrics view', () => {
+  let store: MockStore<State>;
+
   beforeEach(async () => {
     await TestBed.configureTestingModule({
-      declarations: [MetricsDashboardContainer],
-      providers: [],
+      declarations: [MetricsDashboardContainer, RunsSelectorContainer],
+      providers: [provideMockTbStore()],
       // Ignore errors from components that are out-of-scope for this test:
       // 'runs-selector'.
       schemas: [NO_ERRORS_SCHEMA],
     }).compileComponents();
+
+    store = TestBed.inject<Store<State>>(Store) as MockStore<State>;
   });
 
   it('renders', () => {
+    store.overrideSelector(getEnableHparamsInTimeSeries, false);
```

Comment:
Not a big deal since these are the only 2 tests but, this should probably be in the beforeEach.

Reply:
Since this is not always the same I would actually prefer to keep it overridden here for now.